public/app/features/admin/AdminEditOrgPage.tsx

```diff
@@ -52,23 +58,31 @@ export const AdminEditOrgPage: FC<Props> = ({ match }) => {
     return await getBackendSrv().put('/api/orgs/' + orgId, { ...orgState.value, name });
   };
 
+  const renderMissingUserListRightsMessage = () => {
+    return (
+      <Alert severity="info" title="Missing permissions">
+        You do not have permission to see users in this organization. To update this organization, contact your server
```

Comment:
```suggestion
        You do not have permission to see users in this organization. To update this organization, contact your server
```
Does this message have to be this specific? Could we just have the "access denied" as the title and a short description suggesting to contact administrator for getting relevant permissions.

Reply:
I'd prefer it to be specific for the sake of consistency with what we've done so far.